Source/WebKit/WPEPlatform/wpe/WPEToplevel.cpp

```diff
@@ -126,6 +175,29 @@ void wpeToplevelRemoveView(WPEToplevel* toplevel, WPEView* view)
     toplevel->priv->views.remove(view);
 }
 
+GRefPtr<WPEView> wpeToplevelGetView(WPEToplevel* toplevel, size_t index)
+{
+    if (index >= toplevel->priv->views.size())
+        return nullptr;
```

Comment:
Why would the index ever be out of bounds? Shouldn't this be a `RELEASE_ASSERT()`?

Reply:
This is called from an ATK interface, we don't want to crash because some AT program called GetChild with a buggy index.